_posts/2019-03-04-jumping-on-the-dash-less-bandwagon.markdown

```diff
@@ -0,0 +1,64 @@
+
+Outside the technical realm, Serverless is also a mindset that flows right back into the core how your business operates. In the next section, I’ll discuss why fully embracing serverless very likely means a shakeup of how IT operates. All traditional IT roles come into question when you consider how much of the day-to-day work that IT people do isn’t directly related to delivering value. Serverless is a great mechanism for identifying waste.
+
+**Link of Interest:** [Contemporary Views on Serverless and Implications](https://m.subbu.org/contemporary-views-on-serverless-and-implications-1c5907c611d8)
+
+## Business Impacts
+
+If you’re non-technical then you are probably wondering why you should care about Serverless. After all, isn't it just another buzzword? Actually, Serverless is going to have more of an impact on how you work and operate your business than it will for the techies implementing it. Speed, visibility and agility is what it’s all about.
+
+Serverless is about speed. In my time at Mechanical Rock, I have personally received an end-to-end feature request (web, backing code and accompanying infrastructure) in the morning and delivered it to production fully tested by lunch time. A by-product of speed is the ability to experiment. Why try out just one idea, when it's possible to try out multiple ideas, gather the results and then implement the best bits?
+
+Serverless grants unrivalled visibility into the cost and profitability of your business at an incredibly fine-grained level. You can literally see how much it costs to provide the ‘Contact us’ form on your website. Don’t believe me? Read the thought-provoking article at the end of this section.
+
+Serverless agility is about the highlighting (and hopefully the removal) of waste. Anything that gets in the way of delivering value is immediately highlighted, so if it is a technical hurdle then go around it; if its operational then work out how to kill it or change it. End customers don’t care about what bureaucracy you think you need, they’ll vote with eyeballs and wallets and just go somewhere else if you can’t deliver the goods.
+
+**Link of Interest:** [Micro-Optimization: Activity-Based Costing for Digital Services?](https://aws.amazon.com/blogs/enterprise-strategy/micro-optimization-activity-based-costing-for-digital-services/)
+
+## Technical Impacts
+
+For all you technical readers out there, be afraid of serverless. It isn’t a flash in the pan buzzword that’s going away. It is coming for you and will make you obsolete if you don’t adapt. The full power of serverless requires a different way of working and thinking.
```

Comment:
I don't think there's any reason to be afraid of it - it provides some level of capital accountability and in many cases makes the job easier - it's a huge opportunity - not something to cower from.

I'd be worried if i was a middling rank-and-file IT person who has to interact constantly with a change management department - or someone with the title "Project Manager".

Reply:
I agree with your second paragraph completely. However I also think that there are a lot of developers out there that will need a good shock to realise that the world has moved on. When unittesting became a thing followed by CI servers, a large number of developers could drag their feet and still be ok. Serverless is coming fast and it just doesn't need the large team of developers that traditional application development does.